lte/gateway/python/integ_tests/defs.mk

```diff
@@ -164,24 +164,32 @@ s1aptests/test_multi_enb_multi_ue_diff_plmn.py \
 s1aptests/test_x2_handover.py \
 s1aptests/test_x2_handover_ping_pong.py \
 s1aptests/test_s1_handover.py \
-s1aptests/test_attach_detach_rar_tcp_data.py \
 s1aptests/test_attach_detach_with_mme_restart.py \
 s1aptests/test_attach_detach_with_mobilityd_restart.py \
 s1aptests/test_idle_mode_with_mme_restart.py \
-s1aptests/test_3485_timer_for_dedicated_bearer_with_mme_restart.py \
-s1aptests/test_3485_timer_for_default_bearer_with_mme_restart.py \
-s1aptests/test_paging_after_mme_restart.py \
 s1aptests/test_attach_nw_initiated_detach_fail.py \
 s1aptests/test_tau_ta_updating.py \
 s1aptests/test_tau_ta_updating_reject.py \
 s1aptests/test_tau_mixed_partial_lists.py \
 s1aptests/test_eps_bearer_context_status_multiple_ded_bearer_deact.py \
 s1aptests/test_guti_attach_with_zero_mtmsi.py \
 s1aptests/test_ics_timer_expiry_with_mme_restart.py \
-s1aptests/test_attach_mobile_reachability_timer_expiry.py \
+s1aptests/test_restore_mme_config_after_sanity.py
+
+EXTENDED_TESTS_NOT_CONTAINER = s1aptests/test_modify_mme_config_for_sanity.py \
+s1aptests/test_3485_timer_for_default_bearer_with_mme_restart.py \
+s1aptests/test_3485_timer_for_dedicated_bearer_with_mme_restart.py \
+s1aptests/test_attach_detach_rar_tcp_data.py \
+s1aptests/test_attach_detach_rar_tcp_he.py \
+s1aptests/test_attach_detach_with_he_policy.py \
+s1aptests/test_paging_after_mme_restart.py \
+s1aptests/test_restore_mme_config_after_sanity.py
+
+EXTENDED_TESTS_LONG = s1aptests/test_modify_mme_config_for_sanity.py \
 s1aptests/test_attach_implicit_detach_timer_expiry.py \
-s1aptests/test_mobile_reachability_tmr_with_mme_restart.py \
 s1aptests/test_implicit_detach_timer_with_mme_restart.py \
```

Comment:
I guess the change in execution order was just random, or is this a stability issue?

Reply:
Yes, no. Only "historic" reasons how this list was obtained. Back to original order.